test/parser.test.js

```diff
@@ -299,14 +299,19 @@ Language: en
     const input = `WEBVTT
 Kind: captions
 Language: en
+X-TIMESTAMP-MAP=LOCAL:00:00:00.000,MPEGTS:0
```

Comment:
Can this be made optional? I don't think it's a good idea to make this always mandatory

Reply:
Not sure what you mean. 

This metadata field is part of the WebVtt spec (see my main PR description for a link to it). 
I haven't made anything "required". My change only prevents the code from breaking when it encounters a metadata line that contains multiple colons, like X-TIMESTAMP does. 

Can you give more details about what you'd like me to make "optional"?